USECASES.md

```diff
@@ -1,5 +1,15 @@
 # Use Cases
 
+## Provisioning Models
```

Comment:
Ignoring Network Peripheral and Network Device seems like you come up with different models depending on how you arange the managment hierarchy.  There are probably more than these 3.  

Not sure if a 4th model where NIC BMC is above the Plat BMC makes sense unless NIC BMC is available in AUX power and is able to control Plat BMC in AUX.

I may make sense to transition between host trusted and host untrusted, for example in a maintenance window controlled by a higher level console.

![image](https://user-images.githubusercontent.com/2252524/179071417-fe0f6bac-efd6-4f06-a3d1-bd3474db1683.png)

Reply:
I am not following these models very well.  I don’t see these as corresponding to any of the models I had written down.  Maybe we can pop into a call to talk through them?

Reply:
Sorry could have explained these better.  I'm coming at this from the prespective of a managment console or onboarding service and assuming a DPU {cores (ARM), networking (NIC), and optional BMC (NIC BMC)} in a server with a platform BMC.  

In a "host not trusted" mode ("infrastructure device"), the host and the infrastructure are logically seperated.  The DPU would be discovered and provisioned independently of host.  The Plat BMC might have limited knowledge (dotted line) of the DPU if it supports NIC system mangment protocols (NC-SI, PLDM, etc.)

In "host trusted" mode ("Data Processor Peripheral"), the host has full privilage to the DPU and can configure the compute or network functions of the DPU directly.  If the DPU has a NIC BMC it would also have control of all the DPU components.  Host and DPU would be provisioned together.

The "one BMC" is where the NIC BMC is put under the control of the platform BMC.  Host and DPU would be provisioned together.